packages/cli/src/commands/index.ts

```diff
@@ -152,6 +152,34 @@ export function registerScriptCommand(program: Command) {
     )
     .action(lazy(() => import('./build').then(m => m.command)));
 
+  command
+    .command('export-dynamic-plugin')
+    .description(
+      'Build and export a plugin package to be loaded as a dynamic plugin. The repackaged dynamic plugin is exported inside a ./dist-dynamic sub-folder.',
+    )
+    .option('--minify', 'Minify the generated code.')
+    .option(
+      '--embed-package [package-name...]',
+      'Optional list of packages that should be embedded inside the generated code of a backend dynamic plugin, removed from the plugin dependencies, while their direct dependencies will be hoisted to the dynamic plugin dependencies.',
+    )
+    .option(
+      '--shared-package [package-name...]',
+      'Optional list of packages that should be considered shared by all dynamic plugins, and will be moved to peer dependencies of the dynamic plugin. The `@backstage` packages are by default considered shared dependencies.',
+    )
+    .option(
+      '--no-install',
+      'Do not run `yarn install` to fill the dynamic plugin `node_modules` folder.',
+    )
+    .option(
+      '--clean',
+      'Remove the dynamic plugin output before exporting again.',
+    )
+    .option(
+      '--dev',
+      'Allow testing/debugging a backend plugin dynamic loading locally. This installs the dynamic plugin content (symlink) into the dynamic plugins root folder configured in the app config. This also creates a link from the dynamic plugin content to the plugin package `src` folder, to enable the use of source maps.',
+    )
+    .action(lazy(() => import('./export-dynamic-plugin').then(m => m.command)));
```

Comment:
I feel these options are relevant to backend and can mostly never be used on the frontend plugins. I wonder if we can make it obvious in any way.

Reply:
I added the ` (backend plugin only)` mention at the end of the description for most of them.
Only `--shared-package` and `--clean` don't have this mention, since I assume they could be meaningful for frontend plugins as well:
- `--clean` can effectively completely remove the `dist-scalprum` folder before doing the export. 
- `--shared-package` may be useful to allow mentioning some packages that you want to be shared in addition to those you already consider so in the generated module federation settings you use)